student.cpp

```diff
@@ -0,0 +1,248 @@
+#include <iostream>
+#include <cstring>
+using namesapce std;
+
```

Comment:
This need to be inside the namespace.

namespace ibonwe {
namespace icecream {